server/models/location.ts

```diff
@@ -1,15 +1,28 @@
 import dynamoose from 'dynamoose';
 
+export enum Tag {
+  WEST = 'west',
+  CENTRAL = 'central',
+  NORTH = 'north',
+  CTOWN = 'ctown', // college town
+  DTOWN = 'dtown', // downtown
+}
```

Comment:
You already defined this enum in `/types/index.ts`

Reply:
Right, `index.ts` is in the frontend folder while `location.ts` is in the server folder. I'm not sure if there's a way to avoid redefining a type. Is there?